gateway/ln-gateway/src/rpc/rpc_client.rs

```diff
@@ -36,65 +40,65 @@ impl GatewayRpcClient {
     }
 
     pub async fn get_info(&self) -> GatewayRpcResult<GatewayInfo> {
-        let url = self.base_url.join("/info").expect("invalid base url");
+        let url = self.base_url.join(GATEWAY_INFO).expect("invalid base url");
         self.call_get(url).await
     }
 
     // FIXME: deprecated >= 0.3.0
     pub async fn get_info_legacy(&self) -> GatewayRpcResult<GatewayInfo> {
-        let url = self.base_url.join("/info").expect("invalid base url");
+        let url = self
+            .base_url
+            .join(HANDLE_POST_INFO)
+            .expect("invalid base url");
         self.call_post(url, ()).await
     }
 
     pub async fn get_config(&self, payload: ConfigPayload) -> GatewayRpcResult<GatewayFedConfig> {
-        let url = self.base_url.join("/config").expect("invalid base url");
+        let url = self.base_url.join(CONFIGURATION).expect("invalid base url");
```

Comment:
You need to change these on the server side too, not just the gateway. You'll also need to change them inside the lightning client. Do a `grep` for each of these strings to find where they're currently being used, then run the tests to make sure nothing is broken and the client can still reach the server's endpoints.

Reply:
Done. Would it be necessary to replace all occurrences of the strings in the project?

Reply:
You have some failing tests like `test_can_change_routing_fees`. Try to debug those tests and find the locations where you need to update the strings.

Reply:
Done